sktime/utils/_testing/estimator_checks.py

```diff
@@ -426,3 +429,31 @@ def check_persistence_via_pickle(Estimator):
     for method in results:
         unpickled_result = getattr(unpickled_estimator, method)(*args[method])
         _assert_almost_equal(results[method], unpickled_result)
+
+
+def check_multiprocessing_determinism(Estimator):
+    if 'n_jobs' in signature(Estimator.__init__).parameters:
+        estimator = _construct_instance(Estimator)
+        fit_args = _make_args(estimator, "fit")
+        ops = ["predict", "predict_proba", "transform"]
+        for op in ops:
+            if hasattr(estimator, op):
+                X_test = _make_args(estimator, op)[0]
```

Comment:
It's not always `X_test`, I'd just call it args (e.g. may be the forecasting horizon for forecasters), just to avoid confusion.

Reply:
Renamed this.